### PR TITLE
skip quantizing ops in cpu inference

### PR DESCRIPTION
**Pull request by juncaipeng**
### PR types

Others
### PR changes

Others
### Describe

* skip quantizing ops in cpu inference

* remove fake_channel_wise_quantize_dequantize_abs_max to support per-channel quantization for dygraph quantized model

### CODE REVIEW

**Comment by juncaipeng** [comment]
@wojtuss @lidanqing-intel  Please review this PR, thank you.

File: paddle/fluid/framework/ir/mkldnn/cpu_quantize_placement_pass.cc [hunk 1]
```diff
@@ -42,6 +42,10 @@ void CPUQuantizePlacementPass::ApplyImpl(ir::Graph* graph) const {
       return;
     }
 
+    if (op->Op()->GetAttrIfExists<int>("skip_quant") == 1) {
```
Comment (wojtuss) [inline paddle/fluid/framework/ir/mkldnn/cpu_quantize_placement_pass.cc:45]: The attribute is set to `True`, so I would get its value as `bool`.
Apart from that LGTM.

Comment (juncaipeng) [reply paddle/fluid/framework/ir/mkldnn/cpu_quantize_placement_pass.cc:45]: I have set its value as bool, but it raises error. I think Paddle framework changes the data type from bool to int.

The skip_quant attr in saved model as follows.

image

Comment (wojtuss) [reply paddle/fluid/framework/ir/mkldnn/cpu_quantize_placement_pass.cc:45]: OK, that's fine then.

**Review by wojtuss — APPROVED** [review]
LGTM

**Pull request merged**